joblibspark/backend.py

```diff
@@ -202,11 +208,18 @@ def mapper_fn(_):
         except ImportError:
             pass
 
+        if SafeFunction is None:
+            # pylint: disable=protected-access,used-before-assignment
+            return self._get_pool().apply_async(
+                PoolManagerMixin._wrap_func_call, (run_on_worker_and_fetch_result,),
+                callback=callback, error_callback=callback
+            )
```

Comment:
```suggestion
            pool = self._get_pool()
            return .apply_async(
                pool._wrap_func_call, (run_on_worker_and_fetch_result,),
                callback=callback, error_callback=callback
            )
```

We don't need to import `PoolManagerMixin`

Reply:
I may have missed something, but the variable `pool` is an instance of the `ThreadPool` class and does not have the `_wrap_func_call` method. Do you suggest adding the `_wrap_func_call` method to the `SparkDistributedBackend` class instead of importing it from `PoolManagerMixin`?